prac_06/languages.py

```diff
@@ -0,0 +1,13 @@
+from programming_language import ProgrammingLanguage
+
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+
+languages = [python, ruby, visual_basic]
+print("The dynamically typed languages are:")
+for language in languages:
+    if language.typing == "Dynamic":
```

Comment:
it would be better to use the is_dynamic() method instead of directly comparing the 'typing' attribute when filtering dynamic languages.